arangod/Cluster/ActionDescription.cpp

```diff
@@ -56,6 +56,11 @@ std::string ActionDescription::get(std::string const& p) const {
   return _description.at(p);
```

Comment:
Wait a minute ... this can throw.  Is that the proper response?

Reply:
And should be caught. updated documentation,.
Result ActionDescription::get(std::string const& p, std::string& r) does not throw. Should be used, if not checked for existence before.